test/integration/params-validations-test.js

```diff
@@ -127,4 +127,24 @@ describe('params validations', () => {
       due_on: new Date('2012-10-09T23:39:01Z')
     })
   })
+
+  it('Date is passed in correct format for notifications', () => {
+    const github = new GitHub({
+      host: 'notifications-test-host.com'
```

Comment:
I saw this pattern used elsewhere of setting a host in order to locally mock the http requests. Not sure if this is the preferred technique, but I added it to get the tests working

Reply:
Thanks 👌 It is to prevent conflicts with [nock](https://github.com/node-nock/nock) mocks from other test files